.gitignore

```diff
@@ -7,3 +7,5 @@ truffle/combined-sol
 truffle/build
 truffle/npm-debug.log
 truffle/abi/
+meta-contracts/
```

Comment:
Remove these entries to gitignore please, I don't think it's going to be a typical use case to have meta-contracts cloned inside smart-contracts

Reply:
Right now, I have the meta-contracts cloned inside smart-contracts... Because I used a script very similar to the one in circleci to locally generate meta.json. That's the other script in gitignore. I guess I can move the location of the cloned meta-contracts to the parent folder by updating the script.